services/brig/src/Brig/Options.hs

```diff
@@ -563,14 +566,17 @@ defSftServiceName = "_sft"
 defSftDiscoveryIntervalSeconds :: DiffTime
 defSftDiscoveryIntervalSeconds = secondsToDiffTime 10
 
+defSftListLength :: Range 1 100 Int
+defSftListLength = unsafeRange 5
+
 instance FromJSON Timeout where
   parseJSON (Y.Number n) =
     let defaultV = 3600
         bounded = toBoundedInteger n :: Maybe Int64
-     in pure $
-          Timeout $
-            fromIntegral @Int $
-              maybe defaultV fromIntegral bounded
+     in pure
+          $ Timeout
+          $ fromIntegral @Int
+          $ maybe defaultV fromIntegral bounded
```

Comment:
I think you're not using the right version of ormolu.

Reply:
Fixed.